src/tests/functional/inference_engine/CMakeLists.txt

```diff
@@ -16,6 +17,7 @@ set(LINK_LIBRARIES
     openvino::conditional_compilation
     sharedTestClasses
     inference_engine_snippets
+    snippetsNgraphFunctions
```

Comment:
Duplication

Reply:
Removed inference_engine_snippets from the LINK_LIBRARIES